cPP_DSC.adoc

```diff
@@ -193,7 +193,7 @@ DSCs provide seven core security services to a platform as illustrated in <<Core
 |The DSC shall create SDOs from parsed or generated SDEs and attributes using binding mechanisms to apply integrity protection to the SDEs together with their attributes.
 
 |Protect
-|The DSC shall manage protected storage for all SDOs. Platform users (through client applications) can store SDOs either inside or outside the DSC boundary. A DSC shall maintain the integrity and confidentiality (if required) of SDOs both inside the boundary and stored outside the boundary.
+|The DSC shall manage protected storage for all SDOs. Different DSCs may implement local storage internal to the DSC boundary or utlilize external storage outside the DSC boundary. A DSC shall maintain the integrity and confidentiality (if required) of SDOs both inside the boundary and stored outside the boundary.
```

Comment:
I'd suggest to replace "SDOs both inside the boundary and stored outside the boundary" with "SDOs stored both inside and outside the boundary."